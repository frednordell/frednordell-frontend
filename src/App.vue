<script setup lang="ts">
import Footer from './components/Footer.vue';
</script>

<template>
  <div>
    <img src="./assets/face.webp" class="headshot" alt="Picture of Fred">
  </div>
  <h1>Fred Nordell</h1>
  <p>
    Counsultant at <a href="https://netlight.com" target="_blank">Netlight</a>
  </p>
  <p class="reach-out">Reach out at <a href="mailto:fred@nordells.nu">fred@nordells.nu</a></p>
  <Footer></Footer>
</template>

<style scoped>
.headshot {
  height: 16em;
  padding: 1.5em;
  border-radius: 14%;
}
.reach-out {
  color: #888;
}
</style>
