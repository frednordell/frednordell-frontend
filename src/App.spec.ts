import "@vitest/browser/matchers.d.ts";
import { expect, test } from 'vitest'
import { render } from 'vitest-browser-vue'
import App from './App.vue'

test('should render name', async () => {
  const { getByText } = render(App)

  await expect.element(getByText('Fred Nordell')).toBeInTheDocument()
})

test("should match snapshot", async () => {
  const app = render(App);

  expect(app).toMatchSnapshot()
})