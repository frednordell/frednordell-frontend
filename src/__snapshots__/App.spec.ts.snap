// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should match snapshot 1`] = `
{
  "baseElement": <body>
    
    
    
  


    <div />
    <div>
      
      <div
        data-v-7a7a37b1=""
      >
        <img
          alt="Picture of Fred"
          class="headshot"
          data-v-7a7a37b1=""
          src="/src/assets/face.webp"
        />
      </div>
      <h1
        data-v-7a7a37b1=""
      >
        Fred Nordell
      </h1>
      <p
        data-v-7a7a37b1=""
      >
         Counsultant at 
        <a
          data-v-7a7a37b1=""
          href="https://netlight.com"
          target="_blank"
        >
          Netlight
        </a>
      </p>
      <p
        class="reach-out"
        data-v-7a7a37b1=""
      >
        Reach out at 
        <a
          data-v-7a7a37b1=""
          href="mailto:fred@nordells.nu"
        >
          fred@nordells.nu
        </a>
      </p>
      <footer
        data-v-4896eafd=""
        data-v-7a7a37b1=""
      >
        <a
          data-v-4896eafd=""
          href="https://github.com/frednordell"
          target="_blank"
        >
          <img
            alt="Github Logo"
            class="footer-logo"
            data-v-4896eafd=""
            src="/src/assets/GitHub-Mark-Light-64px.png"
          />
        </a>
        <a
          data-v-4896eafd=""
          href="https://www.linkedin.com/in/frednordell"
          target="_blank"
        >
          <img
            alt="LinkedIn Logo"
            class="footer-logo"
            data-v-4896eafd=""
            src="/src/assets/In-White-66px-R.png"
          />
        </a>
      </footer>
      
    </div>
  </body>,
  "container": <div>
    
    <div
      data-v-7a7a37b1=""
    >
      <img
        alt="Picture of Fred"
        class="headshot"
        data-v-7a7a37b1=""
        src="/src/assets/face.webp"
      />
    </div>
    <h1
      data-v-7a7a37b1=""
    >
      Fred Nordell
    </h1>
    <p
      data-v-7a7a37b1=""
    >
       Counsultant at 
      <a
        data-v-7a7a37b1=""
        href="https://netlight.com"
        target="_blank"
      >
        Netlight
      </a>
    </p>
    <p
      class="reach-out"
      data-v-7a7a37b1=""
    >
      Reach out at 
      <a
        data-v-7a7a37b1=""
        href="mailto:fred@nordells.nu"
      >
        fred@nordells.nu
      </a>
    </p>
    <footer
      data-v-4896eafd=""
      data-v-7a7a37b1=""
    >
      <a
        data-v-4896eafd=""
        href="https://github.com/frednordell"
        target="_blank"
      >
        <img
          alt="Github Logo"
          class="footer-logo"
          data-v-4896eafd=""
          src="/src/assets/GitHub-Mark-Light-64px.png"
        />
      </a>
      <a
        data-v-4896eafd=""
        href="https://www.linkedin.com/in/frednordell"
        target="_blank"
      >
        <img
          alt="LinkedIn Logo"
          class="footer-logo"
          data-v-4896eafd=""
          src="/src/assets/In-White-66px-R.png"
        />
      </a>
    </footer>
    
  </div>,
  "debug": [Function],
  "emitted": [Function],
  "getByAltText": [Function],
  "getByLabelText": [Function],
  "getByPlaceholder": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
